SQL/New_patches/2019-02-27_add_biobank_module_to_schema.sql

```diff
@@ -0,0 +1,508 @@
+  `Label` varchar(50) NOT NULL,
+  CONSTRAINT `PK_biobank_specimen_process` PRIMARY KEY (`SpecimenProcessID`),
+  CONSTRAINT `UK_biobank_specimen_process_Label` UNIQUE (`Label`)
+);
+
+CREATE TABLE `biobank_specimen_protocol` (
+  `SpecimenProtocolID` integer unsigned NOT NULL AUTO_INCREMENT,
+  `Label` varchar(50) NOT NULL,
+  `SpecimenProcessID` integer unsigned NOT NULL,
+  `SpecimenTypeID` integer unsigned NOT NULL,
+  CONSTRAINT `PK_biobank_specimen_protocol` PRIMARY KEY (`SpecimenProtocolID`),
+  CONSTRAINT `FK_biobank_specimen_protocol_SpecimenProcessID`
+    FOREIGN KEY (`SpecimenProcessID`) REFERENCES `biobank_specimen_process`(`SpecimenProcessID`)
+    ON UPDATE RESTRICT ON DELETE RESTRICT,
+  CONSTRAINT `FK_biobank_specimen_protocol_SpecimenTypeID`
+    FOREIGN KEY (`SpecimenTypeID`) REFERENCES `biobank_specimen_type`(`SpecimenTypeID`)
+    ON UPDATE RESTRICT ON DELETE RESTRICT,
+  CONSTRAINT `UK_biobank_specimen_protocol_Label` UNIQUE (`Label`)
+) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4; 
+
+CREATE TABLE `biobank_specimen` (
+  `SpecimenID` integer unsigned NOT NULL AUTO_INCREMENT,
+  `ContainerID` integer unsigned NOT NULL, /*Index by ContainerID*/
+  `SpecimenTypeID` integer unsigned NOT NULL,
+  `Quantity` DECIMAL(10, 3) NOT NULL,
+  `UnitID` integer unsigned NOT NULL,
+  `SessionID` integer unsigned NOT NULL,
+  CONSTRAINT `PK_biobank_specimen` PRIMARY KEY (`SpecimenID`),
+  CONSTRAINT `FK_biobank_specimen_ContainerID`
+    FOREIGN KEY (`ContainerID`) REFERENCES `biobank_container`(`ContainerID`)
+    ON UPDATE RESTRICT ON DELETE RESTRICT,
+  CONSTRAINT `FK_biobank_specimen_SpecimenTypeID` 
+    FOREIGN KEY (`SpecimenTypeID`) REFERENCES `biobank_specimen_type`(`SpecimenTypeID`)
+    ON UPDATE RESTRICT ON DELETE RESTRICT,
+  CONSTRAINT `FK_biobank_specimen_UnitID` 
+    FOREIGN KEY (`UnitID`) REFERENCES `biobank_unit` (`UnitID`)
+    ON UPDATE RESTRICT ON DELETE RESTRICT,
+  CONSTRAINT `FK_biobank_specimen_SessionID`
+    FOREIGN KEY (`SessionID`) REFERENCES `session`(`ID`)
+    ON UPDATE RESTRICT ON DELETE RESTRICT,
+  CONSTRAINT `UK_biobank_specimen_ContainerID` UNIQUE (`ContainerID`)
+) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4;
+
+CREATE TABLE `biobank_specimen_freezethaw` (
+  `SpecimenID` integer unsigned NOT NULL,
+  `FreezeThawCycle` integer unsigned NOT NULL,
+  CONSTRAINT `PK_biobank_specimen_freezethaw` PRIMARY KEY (`SpecimenID`),
+  CONSTRAINT `FK_biobank_specimen_freezethaw_SpecimenID`
+    FOREIGN KEY (`SpecimenID`) REFERENCES `biobank_specimen`(`SpecimenID`)
+    ON UPDATE RESTRICT ON DELETE RESTRICT
+) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4;
+
+CREATE TABLE `biobank_specimen_collection` (
+  `SpecimenID` integer unsigned NOT NULL,
+  `SpecimenProtocolID` integer unsigned NOT NULL,
+  `Quantity` DECIMAL(10, 3) NOT NULL,
+  `UnitID` integer unsigned NOT NULL,
+  `CenterID` integer unsigned NOT NULL,
+  `ExaminerID` integer unsigned NOT NULL,
+  `Date` DATE NOT NULL,
```

Comment:
Field should not be named 'Date' or 'Time' but with a more descriptive name. 
see https://github.com/aces/Loris/blob/minor/docs/SQLModelingStandard.md
Same for 'biobank_specimen_preparation', 'biobank_specimen_analysis', 'biobank_pool' and maybe others